api/orchestration_movers_test.go

```diff
@@ -53,8 +54,45 @@ func (d *mockDatasync) CancelTaskExecutionWithContext(ctx context.Context, input
 
 var is_running = false
 
+func (d *mockDataSync) ListTaskExecutionsPagesWithContext(ctx context.Context, input *datasync.ListTaskExecutionsInput, callback func(*datasync.ListTaskExecutionsOutput, bool) bool, opts ...request.Option) error {
+
+	out := &datasync.ListTaskExecutionsOutput{
+		TaskExecutions: []*datasync.TaskExecutionListEntry{
+			{TaskExecutionArn: aws.String("arn:aws:datasync:us-east-1:012345678901:task/task-05cd6f77d7b5d15ac/execution/exec-086d6c629a6bf3581")},
+			{TaskExecutionArn: aws.String("arn:aws:datasync:us-east-1:012345678901:task/task-05cd6f77d7b5d15ac/execution/exec-086d6c629a6bf3582")},
+			{TaskExecutionArn: aws.String("arn:aws:datasync:us-east-1:012345678901:task/task-05cd6f77d7b5d15ac/execution/exec-086d6c629a6bf3583")},
+			{TaskExecutionArn: aws.String("arn:aws:datasync:us-east-1:012345678901:task/task-05cd6f77d7b5d15ac/execution/exec-086d6c629a6bf3584")},
+			{TaskExecutionArn: aws.String("arn:aws:datasync:us-east-1:012345678901:task/task-05cd6f77d7b5d15ac/execution/exec-086d6c629a6bf3585")},
+		},
+	}
+
+	callback(out, false)
+	return nil
+
+}
+
+func (d *mockDataSync) DescribeTaskExecutionWithContext(ctx context.Context, input *datasync.DescribeTaskExecutionInput, opts ...request.Option) (*datasync.DescribeTaskExecutionOutput, error) {
+
+	var ret = &datasync.DescribeTaskExecutionOutput{
+		BytesTransferred:         aws.Int64(100),
+		BytesWritten:             aws.Int64(100),
+		EstimatedBytesToTransfer: aws.Int64(100),
+		EstimatedFilesToTransfer: aws.Int64(100),
+		Excludes:                 []*datasync.FilterRule{},
+		FilesTransferred:         aws.Int64(100),
+		Includes:                 []*datasync.FilterRule{},
+		Options:                  &datasync.Options{},
+		Result:                   &datasync.TaskExecutionResultDetail{},
+		StartTime:                aws.Time(time.Now()),
```

Comment:
This time will never match.
You can define a time variable in this package that you use throughout all the tests, so the time is always the same, e.g.
```
        testTime, err := time.Parse("2006-01-02 15:04:05 -0700 MST", "2020-10-09 23:55:57 +0000 UTC")
	if err != nil {
		t.Errorf("failed to parse time: %s", err)
	}
```
Then here you can do:
```
StartTime: aws.Time(testTime),
```

Reply:
Done..